pkg/manifest/service.go

```diff
@@ -335,13 +342,11 @@ func (sr ServiceResource) GetConfigMapKey() string {
 	return DEFAULT_RESOURCE_ENV_NAME
 }
 
-// skipcq
 func (s Service) AnnotationsMap() map[string]string {
 	annotations := map[string]string{}
 
 	for _, a := range s.Annotations {
-		parts := strings.SplitN(a, "=", 2)
```

Comment:
Ahh, I just upgraded Convox to the latest stable version and was wondering why my proxy-buffer-size annotations dissappeared!

```
    ingressAnnotations:
      # Required for link headers on homepage for preloading assets
      - nginx.ingress.kubernetes.io/proxy-buffer-size=32k
```

so this needs to be key/value now, like this?

```
    ingressAnnotations:
      # Required for link headers on homepage for preloading assets
      - nginx.ingress.kubernetes.io/proxy-buffer-size: 32k
```

Reply:
Ahh, I got it work again by renaming `ingressAnnotations` to `annotations`. Sorry I missed that:

```
services:
  web:
    annotations:
      - nginx.ingress.kubernetes.io/proxy-buffer-size: "32k"
```